### ISSUE TITLE
Codemod should refactor "dynamic" imports to next/dynamic

### ISSUE THREAD

**Issue by tmcw** [blitz-js/blitz#3477]
### What is the problem?

Right now when I transition a codebase into the blitz/toolkit era, an existing import from `blitz` of the `dynamic` method is not rewritten. I think this should be, to `next/dynamic`. As far as I can tell, the blitz-style dynamic method is no longer viable.

### Paste all your error logs here:

```
TypeError: (0 , blitz__WEBPACK_IMPORTED_MODULE_2__.dynamic) is not a function
  17 |   modal: dialogState.DialogStateImport;
  18 |   onClose: () => void;
> 19 | }>(() => import("components/dialogs/import").then((r) => r.ImportDialog), {
     |   ^
  20 |   loading: () => <Loading />,
  21 | });
  22 |

```

### Paste all relevant code snippets here:

```
import { dynamic } from "blitz";
```

### What are detailed steps to reproduce this?

1. Have an existing codebase that uses dynamic
2. Run the blitzjs/codemod.

### Run `blitz -v` and paste the output here:

```
Loaded env from /Users/tmcw/p/placemark/.env.development
Loaded env from /Users/tmcw/p/placemark/.env.local
macOS Monterey | darwin-x64 | Node: v16.13.1

blitz: 0.45.2 (global)
blitz: 0.45.4 (local)

  Package manager: yarn
  System:
    OS: macOS 12.4
    CPU: (16) x64 Intel(R) Core(TM) i9-9880H CPU @ 2.30GHz
    Memory: 185.29 MB / 16.00 GB
    Shell: 5.9 - /usr/local/bin/zsh
  Binaries:
    Node: 16.13.1 - ~/n/bin/node
    Yarn: 1.22.4 - ~/.yarn/bin/yarn
    npm: 6.14.15 - /usr/local/bin/npm
    Watchman: Not Found
  npmPackages:
    @prisma/client: latest => 3.15.2
    blitz: alpha => 0.45.4
    prisma: latest => 3.15.2
    react: latest => 18.2.0
    react-dom: latest => 18.2.0
    typescript: latest => 4.6.4)
```

**Comment by beerose** [comment]
Hey! So we have it in codemod https://github.com/blitz-js/blitz/blob/fix-rpc-log/packages/codemod/src/upgrade-legacy.ts#L166, but maybe there's a case that we don't cover. Could you paste a file or a part of a file where the import was supposed to be updated?

**Comment by tmcw** [comment]
I think the issue here is that I have a top-level folder called `components`, and the automatic refactoring hasn't modified any of the files in that folder. This might be a dontfix and really I should just move that folder into `app/`.

**Comment by dillonraphael** [comment]
Fixed in #3509
